lib/child_process.js

```diff
@@ -118,18 +120,26 @@ exports.execFile = function(file /*, args, options, callback*/) {
     env: null
   };
 
-  // Parse the parameters.
+  // Parse the optional positional parameters.
+  var pos = 1;
+  if (pos < arguments.length && Array.isArray(arguments[pos])) {
+    args = arguments[pos++];
+  } else if (pos < arguments.length && arguments[pos] == null) {
+    pos++;
+  }
```

Comment:
Technically. But this is already in v0.12 and is part of the convergence work. It will land in master and will be cherry picked to v4.x